Sample.App.Database/Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM mcr.microsoft.com/dotnet/core/sdk:3.1

```

Comment:
I think it would be faster if you use volumes and just run `dotnet run` - unless there's an underlying condition that requires publishing the app first?

Reply:
I introduced the Docker file to benefit from Docker caching layers on successive `docker compose up`.
Otherwise, the app gets restored and compiled every time.